server/server.js

```diff
@@ -63,6 +63,7 @@ app.prepare()
         // use routes
         server.use(userInViews());
         server.use('/api/users', users);
```

Comment:
all your routes should handled in your `./routes/index.js`, including your api routes

Reply:
Jude originally created the User API, I have used this convention for the other APIs I created. I will refactor jude User API in another PR

Reply:
sounds good